src/OrchardCore/OrchardCore.Localization.Core/PortableObject/PortableObjectStringLocalizer.cs

```diff
@@ -182,7 +182,7 @@ private string GetTranslation(string name, string context, CultureInfo culture,
                 var translation = dictionary[key, count];
 
                 // Should we search in the parent culture?
```

Comment:
We should search in the parent culture(s) if the `FallBackToParentCulture` is `true`

Reply:
I may push another PR to support culture hierarchy fallback, with context in mind in each call